Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H

```diff
@@ -125,161 +125,168 @@ void UpdateMomentumPerezElastic (
     }
 
     // Compute s
-    const auto tts = m1*g1s*m2*g2s/(inv_c2*p1sm*p1sm) + T_PR(1.0);
-    const auto tts2 = tts*tts;
-    T_PR s = n1*n2/n12 * dt*lnLmd*q1*q1*q2*q2 /
-          ( T_PR(4.0) * MathConst::pi * PhysConst::ep0 * PhysConst::ep0 *
-            m1*g1*m2*g2/(inv_c2*inv_c2) ) * gc*p1sm/mass_g * tts2;
+    T_PR s = 0;
+    if (p1sm > std::numeric_limits<T_PR>::min()) {
```

Comment:
Is there a potential issue for the `b0` parameter above, too? There is a division by p1sm^2 above.

Reply:
That's a good point. I will update this accordingly.